packages/jest-snapshot/tsconfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "build",
+    "importHelpers": true,
+    "allowSyntheticDefaultImports": true
+  },
+  "references": [
```

Comment:
this is missing `jest-diff`, `jest-matcher-utils` and `jest-resolve`

Reply:
I did not understand Typesciprt Project reference well 🙈.
It is needed only for generating `d.ts` files that it depends on?

Reply:
Heh, understandable!

`references` is to tell `tsc` about depencies, so that it builds things in the correct order. So it won't try to build this module before its dependencies have had their type info emitted.

If https://github.com/Microsoft/TypeScript/issues/25376 had been implemented, `yarn` would have taken care of this for us - for now we have to be explicit